### ISSUE TITLE
Single page: sidebars are not recognized correctly in Google Chrome accessibility tree

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/Learn#1959]
## Description

The sidebar uses an `aside` element wrapped in and `article` and `section`. Because of this the role is set to `generic` instead of `complementery` due to a [Chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1459657#c12).

A fix was attempted in https://github.com/WordPress/Learn/pull/1644 but was unsuccessful.

Suggested solution: https://wordpress.slack.com/archives/C01KC5VDQBC/p1698861382115899

## Step-by-step reproduction instructions

1. Go to a single page, eg. https://learn.wordpress.org/lesson-plan/how-to-create-a-lesson-plan/
2. Inspect the right sidebar and find the `aside` element
3. Check the Role in the accessibility panel

## Expected behaviour
The element should have Role="complementary"

## Actual behaviour
The element has Role="generic"

## Environment information
- Chromium browsers

**Comment by adamwoodnz** [comment]
Fixed with the launch of the 2024 block theme
